runtime/src/qio/qbuffer.c

```diff
@@ -96,6 +97,23 @@ void debug_print_bytes(qbytes_t* b)
           b->free_function, b->flags);
 }
 
+void debug_print_iovec(const struct iovec* iov, int iovcnt, size_t maxbytes)
+{
+  int i;
+  size_t j;
+  size_t nb = 0;
+  for( i = 0; i < iovcnt; i++ ) {
+    for( j = 0; j < iov[i].iov_len && nb < maxbytes; j++,nb++) {
+      char* buf = (char*) iov[i].iov_base;
+      char c = '.';
+      if (isprint(buf[j]))
+        c = buf[j];
+      printf("%c", c);
+    }
+  }
+  printf("\n");
+}
+
```

Comment:
To be clear - debug_print_iovec is never actually called (it's just there so I don't have to write it again next time I'm debugging such a thing).